src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Colors getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
+        return Colors.values()[index];
```

Comment:
1. Please extract new Random() as class variable not to create new object each time method us called
2. Method return type should be changed to work properly with constructor of Ball class
```suggestion
    private final Random random = new Random();

    public String getRandomColor() {
        int index = random.nextInt(Colors.values().length);
        return Color.values()[index].name();
```